### PR TITLE
switch to libera.chat

### PR DESCRIPTION
**Pull request by ArneBab**

### CODE REVIEW

**Comment by nextgens** [comment]
I'm not against the idea... but the server should be irc.libera.chat and it would probably make sense to use ircs:// (and specify a port number)

**Comment by ArneBab** [comment]
Fixed — thank you!

**Review by Bombe — APPROVED** [review]
Yeah, let's move away from the shitfest that freenode has been turned into as fast as possible.

File: content/pages/help.md [hunk 1]
```diff
@@ -602,7 +602,7 @@ download page for a specific build, or use git to get an up to date copy of
 the source, see [here](https://wiki.freenetproject.org/Development) for details. Further
 instructions for building and deploying the server are included with the code
 itself. Generally speaking, joining our IRC channel is a good idea: [#freenet
-on chat.freenode.net](irc://chat.freenode.net/freenet)
+on libera.chat](rcs://irc.libera.chat:6697/freenet)
```
Comment (Thynix) [inline content/pages/help.md:605]: Looks like `rcs` should be `ircs`?

Comment (ArneBab) [reply content/pages/help.md:605]: arg, yes!

Comment (ArneBab) [reply content/pages/help.md:605]: See https://github.com/freenet/website/commit/e4caa1cfd604a11d9c2218f72ad0b88f2bb2f5fb

**Comment by Thynix** [comment]
... Has this been deployed?

**Comment by Thynix** [comment]
@ArneBab This didn't get deployed - https://freenetproject.org/pages/help.html still contains

> Many of the developers and users of Freenet are on the IRC channel #freenet on chat.freenode.net.

**Comment by ArneBab** [comment]
Our website deployment travis action is broken (by travis) and not yet converted to github actions.

**Pull request merged**